base.js

```diff
@@ -30,5 +30,10 @@ module.exports = {
     'prefer-template': 'error',
     'prefer-object-spread': 'error',
     'comma-dangle': 'off',
+    camelcase: 'error',
+    eqeqeq: 'error',
```

Comment:
not sure that this is needed. != null useful sometimes, no cases when it was harmful yet

Reply:
We can use this https://eslint.org/docs/rules/eqeqeq#allow-null or this https://eslint.org/docs/rules/eqeqeq#smart instead.

Reply:
let's go with https://eslint.org/docs/rules/eqeqeq#smart